src/android/ContactAccessorSdk5.java

```diff
@@ -451,7 +452,13 @@ else if (mimetype.equals(CommonDataKinds.Website.CONTENT_ITEM_TYPE)
                     else if (mimetype.equals(CommonDataKinds.Event.CONTENT_ITEM_TYPE)) {
                         if (isRequired("birthday", populate) &&
                                 CommonDataKinds.Event.TYPE_BIRTHDAY == c.getInt(colEventType)) {
-                            contact.put("birthday", c.getString(colBirthday));
+
+                            try {
+                                long timestamp = Date.valueOf(c.getString(colBirthday)).getTime();
+                                contact.put("birthday", timestamp);
+                            } catch (IllegalArgumentException e) {
+                                Log.d(LOG_TAG, "Failed to get birthday for contact: " + e.getMessage());
+                            }
```

Comment:
> Also, if there is an IllegalArgumentException, a birthday won't be set at all; is this what we define as expected behaviour for this plugin's API?

I don't see any other appropriate options here. If there is an exception, this means that the date is probably malformed, and we can't convert it into a valid timestamp, which JS side expects to get.